environments/swiss/public.yml

```diff
@@ -129,8 +129,10 @@ localsettings:
   #  PILLOWTOP_MACHINE_ID:
   LOCAL_MIDDLEWARE:
     - 'django.middleware.security.SecurityMiddleware'
-  SECURE_HSTS_SECONDS: "{{ nginx_hsts_max_age }}"
   SILENCED_SYSTEM_CHECKS:
```

Comment:
since these apply globally to all envs it might make more sense to set it directly in the localsetting template instead of having to copy paste in all env files.

Reply:
Yeah that is a good idea, and I just made that change. When you say _all envs_, that is not entirely true right? I don't have great context on the remaining envs outside of staging, swiss, india, and prod, but do we need to take those into consideration?

Reply:
All envs use nginx so they should be covered by the nginx changes (assuming they update their nginx config)

Reply:
But is there any chance that other envs don't enforce HTTPS at all endpoints?

Reply:
I think the approach should be that a default install of CommCare has this set up, either at the nginx level or at the Django level. If a 3rd party is doing something custom then it is up to them to ensure appropriate security. 

Besides that, is there even a way to enable these security features in Django through commcare-cloud?

Reply:
> I think the approach should be that a default install of CommCare has this set up, either at the nginx level or at the Django level. If a 3rd party is doing something custom then it is up to them to ensure appropriate security.

That makes sense. 

> Besides that, is there even a way to enable these security features in Django through commcare-cloud?

I don't fully understand the question. Once using django security middleware, [these settings](https://docs.djangoproject.com/en/3.2/topics/security/#ssl-https) can be used in a `settings.py` file. If your question is asking specifically from a 3rd party perspective, I'm not sure. I'm assuming they would fork commcare-cloud, and in that case have control of security so my initial point on non-https endpoints is moot. Thanks Simon.

Reply:
>> Besides that, is there even a way to enable these security features in Django through commcare-cloud?

> I don't fully understand the question. Once using django security middleware, these settings can be used in a settings.py file. If your question is asking specifically from a 3rd party perspective, I'm not sure. I'm assuming they would fork commcare-cloud, and in that case have control of security

Yea, I was meaning without forking, is there a way to change things with just config in commcare-cloud.